modules/lang-groovy/src/test/java/org/elasticsearch/messy/tests/ExtendedStatsTests.java

```diff
@@ -503,7 +503,7 @@ private void assertShardExecutionState(SearchResponse response, int expectedFail
         ShardSearchFailure[] failures = response.getShardFailures();
         if (failures.length != expectedFailures) {
             for (ShardSearchFailure failure : failures) {
-                logger.error("Shard Failure: {}", failure.reason(), failure.toString());
+                logger.error("Shard Failure: {}", failure);
```

Comment:
Maybe the original author meant `logger.error("Shard Failure: {}", failure.cause(), failure);`?